### ISSUE TITLE
[a11y-defect-3]: Improper use of fieldset and duplicative <legend>

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#64361]
### Point of contact

Jamie Klenetsky Fay @coforma-jamie

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

On [step 2 of the form](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/additional-information), the "What's your level of education?" and "Are you moving in the next 30 days?" fields are contained within a `<fieldset>` with the `<legend>` "Additional information":
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/2a25a5f7-e0bf-43b2-9f34-6290d4ab58fb)

There are two issues with this:
1. These fields don't need to be within a `<fieldset>`. There's only 1 "Additional information" page on this form, and these two questions aren't related to each other, so this extra layer of grouping is superfluous and leads to the second issue:
2. `<legend>Additional Information</legend>` is duplicative of the "Step 2 of 4: Additional Information" heading. A screen reader user would hear something like this: `Heading level 2, Step 2 of 4: Additional information. ... Additional information, group.` This adds more for the SR user to have to parse through unnecessarily.
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/c93ea38d-5d16-4887-8d55-30978ea69a8d)

### Reproduction steps

1. [Go to the form](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)
2. Click "Apply for Veteran Readiness and Employment with VA Form 28-19003" and sign in
4. Fill out the form until you get to [this page](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/additional-information)
5. Inspect the "Additional Information" `<legend>` above the "What's your level of education?" field. You can also use a screen reader like VoiceOver or NVDA to hear what it sounds like.

### Proposed solution or next steps

Take the "education" and "moving" fields out of their grouping `<fieldset>`; remove the `<legend>`. The page will look like this, in pseudocode:

```
<heading>
Step 2 of 4: Additional information
</heading>
....

<select component>
What's your level of education?
..... 
</select>

<radio button component>
Are you moving in the next 30 days?
..... 
</radio>
```

### References, articles, or WCAG support

[When to use a fieldset and legend](https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/#:~:text=When%20to%20use%20a%20fieldset%20and%20legend), from gov.uk

### Type of issue

- [ ] Axe-core
- [X] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [X] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

### Reproduction steps

1. [Go to the form](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/start)
2. Click "Apply for Veteran Readiness and Employment with VA Form 28-19003" and sign in
3. Fill out the form until you get to [this page](https://staging.va.gov/careers-employment/vocational-rehabilitation/apply-vre-form-28-1900/additional-information)
4. Inspect the "Additional Information" `<legend>` above the "What's your level of education?" field. You can also use a screen reader like VoiceOver or NVDA to hear what it sounds like.

### Proposed solution or next steps

Take the "education" and "moving" fields out of their grouping `<fieldset>`; remove the `<legend>`. The page will look like this, in pseudocode:

```
<heading>
Step 2 of 4: Additional information
</heading>
....

<select component>
What's your level of education?
..... 
</select>

<radio button component>
Are you moving in the next 30 days?
..... 
</radio>
```

### References, articles, or WCAG support

[When to use a fieldset and legend](https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/#:~:text=When%20to%20use%20a%20fieldset%20and%20legend), from gov.uk

### Type of issue

- [ ] Axe-core
- [X] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [X] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else
```[tasklist]
### Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/68021
```

**Comment by steele-lm** [comment]
@fiorella-io Do you think this is still an issue with the updated designs?

**Comment by fiorella-io** [comment]
@steele-lm Yes, because we've separated both questions into different pages and steps. 

<img src="https://github.com/user-attachments/assets/f4ff656d-5194-4009-9cda-095c16388f18" width="30%">
<img src="https://github.com/user-attachments/assets/1141fefc-1129-478a-906a-831aba8c1c16" width="30%">
